public/app/core/components/AppChrome/AppChrome.tsx

```diff
@@ -254,10 +274,20 @@ const getStyles = (theme: GrafanaTheme2, headerHeight: number) => {
       },
     }),
     sidebarContainer: css({
-      position: 'fixed',
-      height: `calc(100% - ${headerHeight}px)`,
+      // the `Resizeable` component overrides the needed `position` and `height`
+      // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+      position: 'fixed !important' as 'fixed',
+      height: `calc(100% - ${headerHeight}px) !important`,
       zIndex: 2,
       right: 0,
     }),
   };
 };
+
+const getContentSizeStyles = (_: GrafanaTheme2, extensionSidebarWidth = 0) => {
+  return {
+    contentWidth: css({
+      maxWidth: `calc(100% - ${extensionSidebarWidth}px)`,
+    }),
+  };
+};
```

Comment:
This is needed to fix the `main` width. Especially in dashboards, where the dashboard container has a fixed width, the Resizeable or `useSplitter` was not working properly.